index.js

```diff
@@ -24,39 +25,40 @@ program
       }
     }
 
-    var num = numberOfResults;
-
-    if(numberOfResults === undefined) {
-      num = 3;
-    }
-
     request(options, function (err, res, body) {
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
 
       console.log('='.repeat(consoleWidth()));
 
+      if(typeof process.argv[4] !== 'undefined'){
+        if(typeof Number(process.argv[4]) === 'number'){
+          resultsToDisplay = Number(process.argv[4]);
+        }
+      }
```

Comment:
You're using process.argv[4] 3 times. Can we just bind it to a variable so we don't have to use it 3 times?

Reply:
Actually, you can just put the process.argv into a single if statement
```javascript
if(conditionA && conditionB) {
   // do stuff
}
```